app/views/admin/users/controls/_export.html.erb

```diff
@@ -1,3 +1,3 @@
-<button type="button" class="js-export-csv-modal-trigger c-btn" data-modal-trigger data-modal-title="Download member data" aria-label="Download member data" data-modal-size="small" data-modal-content-selector="#export-csv-modal">
+<button type="button" class="js-export-csv-modal-trigger c-btn" data-modal-title="Download member data" aria-label="Download member data" data-modal-size="small" data-modal-content-selector="#export-csv-modal">
```

Comment:
I've removed `data-modal-trigger` throughout, as we can readily identify if this element is a modal trigger by the `data-modal-content-selector` attribute 🤷‍♀️